styles/app.scss

```diff
@@ -4,3 +4,217 @@
 // $fa-font-path: '//netdna.bootstrapcdn.com/font-awesome/4.3.0/fonts';
 // @import 'font-awesome';
 @import 'box-sizing';
+@import 'reset';
+@import 'twitch';
+
+.tw-app__left {
+  background: #17141f;
+  color: #fff;
+  width: 30%;
+}
+
+.side-bar__left {
+  font-family: Helvetica;
+  background: #17141f;
+  color: #fff;
+  padding: 1rem;
+  width: 100%;
+  position: relative;
+  min-width: 15rem;
+}
+
+.logo {
+  width: 7rem;
+  transform: translate(50%);
+}
+
+.inputAddOn {
+  width: 100%;
+  margin-top: 2rem;
+  display: flex;
+}
+
+.inputAddOn-field {
+  width: 100%;
+  height: 1.9rem;
+  color: #606060;
+  font-size: 1rem;
+  background: #201c2b;
+  border: 3px solid #251f36;
+}
+
+.nav-list {
+  font-size: 0.9rem;
+
+  &__item {
+    padding: 0.3rem 0;
+  }
+}
+
+.channels {
+  border-bottom: 3px solid #251f36;
+  padding-bottom: 0.5rem;
+  font-size: 0.8rem;
+  color: #bab7c2;
+  margin-top: 2rem;
+}
+
+.channels-list__item {
+  display: flex;
+  margin: 0.5rem 0 0;
+
+}
+
+.image-wrap {
+  margin: 0;
+}
+
+.text-wrap {
+  padding: 0;
+  display: flex;
+}
+
+.gamertag {
+  font-size: 1rem;
+  color: #bab7c2;
+}
+
+p {
+  margin: 0;
+  font-size: 0.7rem;
+  color: #8c8083;
+}
+
+.other-list {
+  display: flex;
+  font-size: 0.7rem;
+  color: #606060;
+}
+
+.more-twitch__one {
+  width: 50%;
+}
+
+.more-twitch__two {
+  width: 50%;
+}
+
+.more-twitch__item {
+  padding-bottom: 0.5rem;
+}
+
+.btn-container {
+  display: flex;
+  justify-content: space-around;
+  padding-top: 5rem;
+}
+
+.nav-btn {
+  width: 40%;
+  height: 2rem;
+  border: 2px solid #6441a4;
+  border-radius: 2%;
+  background: #6441a4;
+  color: #fff;
+  font-size: 0.8rem;
+}
+
+.tw-app__center {
+  display: flex;
+  flex-direction: column;
+  min-width: 25rem;
+  width: 85%;
+  margin: 2rem;
+}
+
+.heading {
+  display: flex;
+  justify-content: space-between;
+  height: 8rem;
+  margin: 2rem 0.5rem 0;
+}
+
+.all-games {
+  margin: 0;
+  font-family: Helvetica;
+}
+
+.input {
+  width: 12rem;
+  height: 2rem;
+  border: #efeef1 solid 1px;
+}
+
+.games-list {
+  display: flex;
+  flex-wrap: wrap;
+  flex-direction: row;
+  width: 100%;
+}
+
+.game-item {
+  display: flex;
```

Comment:
If you're putting things into a vertical stack, you don't need to go into flex column mode.

Also, this is leading to some weird stuff since `img` tags REALLY don't like to get flexed.